solana/gatekeeper-lib/src/commands/freeze.ts

```diff
@@ -18,8 +18,8 @@ Frozen
 
   static flags = {
     help: Flags.help({ char: "h" }),
-    gatekeeperKey: gatekeeperKeyFlag(),
-    gatekeeperNetworkKey: gatekeeperNetworkPubkeyFlag(),
+    gatekeeperKeypair: authorityKeypairFlag(),
```

Comment:
this should remain GatekeeperKeypair. Authority only exists on the Network Level not on the Gatekeeper level.